tests/python/relay/test_build_module.py

```diff
@@ -22,48 +22,23 @@
 from tvm import relay
 from tvm.target.target import Target
 from tvm.relay.backend import Runtime, Executor, graph_executor_codegen
-from tvm.relay.build_module import _reconstruct_from_deprecated_options
 
 
-@pytest.mark.parametrize(
-    "target,executor,runtime",
-    [
-        [Target("c"), None, None],
-        [Target("c -runtime=c"), None, Runtime("crt")],
-        [Target("c -system-lib"), None, Runtime("cpp", {"system-lib": True})],
-        [Target("c -runtime=c -system-lib"), None, Runtime("crt", {"system-lib": True})],
-        [Target("c -executor=aot"), Executor("aot"), None],
-        [
-            Target("c -executor=aot -interface-api=c"),
-            Executor("aot", {"interface-api": "c"}),
-            None,
-        ],
-        [
-            Target("c -executor=aot -unpacked-api=1"),
-            Executor("aot", {"unpacked-api": 1}),
-            None,
-        ],
-        [Target("c -executor=aot -link-params=1"), Executor("aot"), None],
-        [Target("c -link-params=1"), Executor("graph", {"link-params": 1}), None],
-        [
-            Target(
-                "c -executor=aot -link-params=1 -interface-api=c"
-                "  -unpacked-api=1 -runtime=c -system-lib"
-            ),
-            Executor("aot", {"unpacked-api": 1, "interface-api": "c"}),
-            Runtime("crt", {"system-lib": True}),
-        ],
-    ],
+test_target = tvm.testing.parameter(
```

Comment:
We should probably keep using `pytest.mark.parametrize` to aid in linting this file.